AWSIoT/AWSIoTManager.m

```diff
@@ -224,6 +224,8 @@ + (BOOL)importIdentityFromPKCS12Data:(NSData *)pkcs12Data passPhrase:(NSString *
 
     if (![AWSIoTKeychain addPrivateKeyRef:privateKey tag:privateTag])
     {
+        if (publicKey)
```

Comment:
Could you please explain why is the public key here? should we release privateKey here?

Reply:
The revealed leak by Xcode clang analyzer is that *publicKey* is not released before we return at this point.

I assume that Private key ownership is given to `[AWSIoTKeychain addPrivateKeyRef:tag:]`, so it would be that method that need to release it if needed.